data/json/terrain.json

```diff
@@ -10690,5 +10690,77 @@
       "ter_set": "t_null",
       "items": [ { "item": "material_soil", "count": [ 8, 20 ] }, { "item": "adobe_brick", "count": [ 2, 6 ] } ]
     }
+  },
+  {
+    "id": "t_leanto",
+    "type": "terrain",
+    "name": "pine lean-to",
+    "symbol": ";",
+    "color": [ "brown" ],
+    "move_cost": 2,
+    "bash": {
+      "str_min": 40,
+      "str_max": 90,
+      "ter_set": "t_dirt",
```

Comment:
This should bash into a pine tree, you shouldn't knock down the whole tree at oncr